pkg/storage/tsdb/config.go

```diff
@@ -156,17 +156,19 @@ func (cfg *Config) Validate() error {
 
 // BucketStoreConfig holds the config information for Bucket Stores used by the querier
 type BucketStoreConfig struct {
-	SyncDir               string           `yaml:"sync_dir"`
-	SyncInterval          time.Duration    `yaml:"sync_interval"`
-	MaxChunkPoolBytes     uint64           `yaml:"max_chunk_pool_bytes"`
-	MaxSampleCount        uint64           `yaml:"max_sample_count"`
-	MaxConcurrent         int              `yaml:"max_concurrent"`
-	TenantSyncConcurrency int              `yaml:"tenant_sync_concurrency"`
-	BlockSyncConcurrency  int              `yaml:"block_sync_concurrency"`
-	MetaSyncConcurrency   int              `yaml:"meta_sync_concurrency"`
-	BinaryIndexHeader     bool             `yaml:"binary_index_header_enabled"`
-	ConsistencyDelay      time.Duration    `yaml:"consistency_delay"`
-	IndexCache            IndexCacheConfig `yaml:"index_cache"`
+	SyncDir                  string           `yaml:"sync_dir"`
+	SyncInterval             time.Duration    `yaml:"sync_interval"`
+	MaxChunkPoolBytes        uint64           `yaml:"max_chunk_pool_bytes"`
+	MaxSampleCount           uint64           `yaml:"max_sample_count"`
+	MaxConcurrent            int              `yaml:"max_concurrent"`
+	TenantSyncConcurrency    int              `yaml:"tenant_sync_concurrency"`
+	BlockSyncConcurrency     int              `yaml:"block_sync_concurrency"`
+	MetaSyncConcurrency      int              `yaml:"meta_sync_concurrency"`
+	BinaryIndexHeader        bool             `yaml:"binary_index_header_enabled"`
+	PostingsCompressions     bool             `yaml:"postings_compression_enabled"`
```

Comment:
This is a duplicate of the setting in the index cache and should be removed.

Reply:
Thanks for catching this! It's a leftover before I decided to move it to IndexCacheConfig.